Sources/PackageLoading/PackageBuilder.swift

```diff
@@ -214,6 +214,7 @@ public struct PackageBuilder {
         } else {
             testModules = []
         }
+        try fillDependencies(modules: modules + testModules)
```

Comment:
the code which did this in `constructModules` is refactored into this method. It wasn't currently possible to call this method individually from both the construct methods but it should be possible to move it done once those two are merged.